codegen/Package.swift

```diff
@@ -164,7 +164,7 @@ func addProtocolTests() {
         )
         let testTarget = protocolTest.buildOnly ? nil : Target.testTarget(
             name: "\(protocolTest.name)Tests",
-            dependencies: [.smithyTestUtils, .smithyStreams, .smithyWaitersAPI, .byNameItem(name: protocolTest.name, condition: nil)],
+            dependencies: [.smithyTestUtils, .smithyStreams, .smithyWaitersAPI, .smithyIdentity, .byNameItem(name: protocolTest.name, condition: nil)],
```

Comment:
`SmithyIdentity` is added as a protocol test target dependency since all tests now use a static credential resolver.